plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -4088,6 +4088,36 @@ class AzureLoadBalancer(AzureResource, BaseLoadBalancer):
     provisioning_state: Optional[str] = field(default=None, metadata={'description': 'The current provisioning state.'})  # fmt: skip
     resource_guid: Optional[str] = field(default=None, metadata={'description': 'The resource GUID property of the load balancer resource.'})  # fmt: skip
     azure_sku: Optional[AzureSku] = field(default=None, metadata={"description": "SKU of a load balancer."})
+    aks_public_ip_address: Optional[str] = field(default=None, metadata={"description": "AKS Load Balancer public IP address."})  # fmt: skip
+
+    def post_process(self, graph_builder: GraphBuilder, source: Json) -> None:
+        def collect_p_ip_address() -> None:
```

Comment:
I still don't get it: the load balancer has a property `lb_frontend_ip_configurations` which points to the public Ip address. Isn't it enough? Why do we need to make another request here?

Reply:
The `publicIPAddress` within the `lb_frontend_ip_configurations` property currently only provide the `publicIPAddress ID`

To get extended version of `publicIPAddress` (with information fields), we need to do request with additional parameters such as `{"$expand": "frontendIPConfigurations/publicIPAddress"}`. However, it's important to note that this request can only be applied to individual resources rather than general resources.

example json without params:
```js
          "frontendIPConfigurations": [
                    {
                        "name": "felb",
                        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/loadBalancers/lb/frontendIPConfigurations/felb",
                        "etag": "W/\"00000000-0000-0000-0000-00000000\"",
                        "type": "Microsoft.Network/loadBalancers/frontendIPConfigurations",
                        "properties": {
                            ...
                              "publicIPAddress": {
                                "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/publicIPAddresses/testDNS-ip",
                                "resourceGroup": "rg1"
                            }
                         }
                    }  
               ]
```
example json with params:
```json
"frontendIPConfigurations": [
     ...
      "properties": {
          ...
           "publicIPAddress": {
                    "name": "testDNS-ip",
                    "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/publicIPAddresses/testDNS-ip",
                    "location": "westus",
                    "properties": {
                      "provisioningState": "Succeeded",
                      "ipAddress": "40.85.154.247",
                      "publicIPAddressVersion": "IPv4",
                      "publicIPAllocationMethod": "Dynamic",
                      "idleTimeoutInMinutes": 4,
                      ...
                    "type": "Microsoft.Network/publicIPAddresses"
                  }
   ]
   ```

Reply:
Exactly. Amd we also collect IP addresses. We should be able to resolve them without an additional http call.